Data/Input/default.xci

```diff
@@ -622,6 +622,14 @@ event=Mode default
 label=Profiles
 location=5
 
+mode=Config2
+type=key
+data=7
+event=UploadIGCFile
+event=Mode default
+label=Weglide
```

Comment:
So thank you for creating the change. 

Do you think the label is sufficient with "Weglide"?